File: api/src/main/java/org/pmiops/workbench/Controller.java
package org.pmiops.workbench;

import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

// XXX: trigger CI - revert.
@RestController
public class Controller {
  @RequestMapping("/")
  public String index() {
    return "AllOfUs Workbench API";
  }
}

File: api/src/main/java/org/pmiops/workbench/Controller.kt
package org.pmiops.workbench

import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class Controller {
    @RequestMapping("/")
    fun index(): String {
        return "AllOfUs Workbench API"
    }
}
